domain/builders/atrium-domain-builders-jvm/src/main/kotlin/ch/tutteli/atrium/domain/builders/creating/PathAssertionsBuilder.kt

```diff
@@ -17,6 +17,9 @@ object PathAssertionsBuilder : PathAssertions {
     override fun <T : Path> endsWith(assertionContainer: Expect<T>, expected: Path) =
         pathAssertions.endsWith(assertionContainer, expected)
 
+    override fun <T : Path> endsNotWith(assertionContainer: Expect<T>, expected: Path) =
```

Comment:
please `inline` the function (could you do the same for endsWith, I apparently did not see it during the review)